var/spack/repos/builtin/packages/amr-wind/package.py

```diff
@@ -43,52 +43,49 @@ class AmrWind(CMakePackage, CudaPackage, ROCmPackage):
         "0.9.0", tag="v0.9.0", commit="cf66ebe31fd5f27b76a83451cd22f346e7a67160", submodules=True
     )
 
-    variant("hypre", default=True, description="Enable Hypre integration")
+    variant("hypre", default=False, description="Enable Hypre integration")
     variant("ascent", default=False, description="Enable Ascent integration")
     variant("masa", default=False, description="Enable MASA integration")
     variant("mpi", default=True, description="Enable MPI support")
-    variant("netcdf", default=True, description="Enable NetCDF support")
+    variant("netcdf", default=False, description="Enable NetCDF support")
     variant("openfast", default=False, description="Enable OpenFAST integration")
     variant("openmp", default=False, description="Enable OpenMP for CPU builds")
     variant("shared", default=True, description="Build shared libraries")
     variant("tests", default=True, description="Activate regression tests")
     variant("tiny_profile", default=False, description="Activate tiny profile")
     variant("hdf5", default=False, description="Enable HDF5 plots with ZFP compression")
-    variant("umpire", default=False, description="Enable Umpire")
+    variant("umpire", default=False, description="Enable UMPIRE memory pooling")
     variant("sycl", default=False, description="Enable SYCL backend")
-    variant("gpu-aware-mpi", default=False, description="gpu-aware-mpi")
+    variant("gpu-aware-mpi", default=False, description="Enable GPU aware MPI")
 
-    depends_on("hypre~int64@2.20.0:", when="+hypre")
-    depends_on("hypre+mpi", when="+hypre+mpi")
+    depends_on("mpi", when="+mpi")
     depends_on("hdf5~mpi", when="+hdf5~mpi")
     depends_on("hdf5+mpi", when="+hdf5+mpi")
     depends_on("h5z-zfp", when="+hdf5")
     depends_on("zfp", when="+hdf5")
-    depends_on("hypre+umpire", when="+umpire")
-    depends_on("hypre+sycl", when="+sycl")
-    depends_on("hypre+gpu-aware-mpi", when="+gpu-aware-mpi")
+    depends_on("hypre~int64@2.20.0:", when="+hypre")
+    depends_on("hypre+mpi", when="+hypre+mpi")
+    depends_on("hypre+umpire", when="+hypre+umpire")
+    depends_on("hypre+sycl", when="+hypre+sycl")
+    depends_on("hypre+gpu-aware-mpi", when="+hypre+gpu-aware-mpi")
     depends_on("hypre@2.29.0:", when="@0.9.0:+hypre")
+    depends_on("masa", when="+masa")
+    depends_on("ascent~mpi", when="+ascent~mpi")
+    depends_on("ascent+mpi", when="+ascent+mpi")
+    depends_on("netcdf-c", when="+netcdf")
+    depends_on("openfast+cxx@2.6.0:3.4.1", when="@0.9.0:1.3.0,main+openfast")
```

Comment:
@michaelkuhn added a `1.3.1` and `1.4.0` releases.  What is the plan for https://github.com/Exawind/amr-wind/pull/1006

@marchdf @michaelkuhn Will this become `amr-wind@2.0`? We should adjust our version semantics here based on that.

Reply:
https://github.com/Exawind/amr-wind/pull/1006 should be merged shortly. There was chatter on Wednesday on if we should hold off but I think we are good. Might do it next early next week to avoid doing anything that might break over the weekend.

Once that PR goes in, that should become 2.0. And then we can update the semantics here to account for the openfast change. 

Does that answer your questions?

Reply:
```suggestion
   when("+openfast"):
      depends_on("openfast+cxx")
      depends_on("openfast@2.6.0:3.4.1", when="@0.9.0:1")
      depends_on("openfast@3.5.1", when="@2:")
```
Perhaps we should just wait for that to merge and do this?

Reply:
yeah. And I would make it `openfast@3.5.3` actually.

Reply:
we could do `openfast@3.5` and then solver should pick the latest patch release.